DirectEmployers/MyJobs
# [PD-3385] tjd-distribution-location-summary

## What's this PR do?
PR updates the distribution location details on the right hand side of TJD manage contact page depending on the rules applied for that contact

## Screenshots (if appropriate)
![screen shot 2017-07-31 at 11 10 37 am](https://user-images.githubusercontent.com/22029077/28784273-e826e1e8-75e0-11e7-9ecd-fbedd3dbd289.png)


## Where should the reviewer start?
Start inside of the TJD application

## How should this be manually tested?
Pull down the repository. Go into TJD. Preferably go into a custom CBO contact. Edit the locations and make some rules and check and make sure the text changes depending on rules applied.

## Any background context you want to provide?
Was merged but needed to update based on Jason Sole requirements
